### ISSUE TITLE
Are you adding Kd_loss to the Total_loss

### ISSUE THREAD

**Issue by faziii0** [RunpeiDong/PointDistiller#13]
I have seen your code; you are not adding your kd_loss to your loss. You are just giving new entry for kd_loss just to show the difference; you are not doing it this way += , loss_kd is **losses['loss_kd'] = kd_loss**. and in your detectors file point_rcnn you just update it **losses.update({'kd_loss': kd_loss})**. you are just adding and updating the dictionary. 

Maybe I didn't understand it right from what i have seen from your code.

Thanks @RunpeiDong

**Comment by RunpeiDong** [comment]
Because that's how mmdet3d calculates losses, please read the code carefully. Once the dict is updated, the loss will be backpropagated. 

Read the `mmdet3d` and `mmdet` codebase first and understand how they work, then you will see. 

**Please Take time and read the code.**

**Comment by faziii0** [comment]
@RunpeiDong sorry to say I cannot see any of the logic or code in pointrcnn or train.py you are adding the kd_loss Can you point out the file name that i have missed from what i am seeing you are getting the kd_loss but their is no implementation to add it to total_loss.
